src/jshint.js

```diff
@@ -863,7 +863,7 @@ var JSHINT = (function() {
       funct["(blockscope)"].stack();
       advance("let");
       advance("(");
-      state.syntax["let"].fud.call(state.syntax["let"].fud, false);
+      state.syntax["let"].fud();
```

Comment:
sorry, I meant I'd prefer `state.tokens.prev.nud()` --- where "prev" is the "let" token at this point